src/ontology/foodon-edit.owl

```diff
@@ -18405,6 +18405,59 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003924 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003924">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001009"/>
+        <obo:IAO_0000115 xml:lang="en">A hot dog on a stick that is coated in a thick layer of cornmeal batter and deep fried.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2000/01/rdf-schema#anyURI">https://en.wikipedia.org/wiki/Corn_dog</obo:IAO_0000119>
+        <obo:IAO_0000232 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000122"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-19T11:58:23Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">corndog</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">corn dog</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003925 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003925">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001173"/>
+        <obo:IAO_0000115>A powder made from chia seeds.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-07-19T12:27:02Z</dc:date>
+        <rdfs:label xml:lang="en">chia seed powder</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003926 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003926">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/STATO_0000503"/>
```

Comment:
"random sample meat cut" should just be a parent of "meat cut" .  The stato term is a sampling strategy, not the material entity that is a piece of meat selected a certain way ;) .